src/domains/apiconfig-app/00_alert_apiconfig_core.tf

```diff
@@ -0,0 +1,74 @@
+#####################
+## API-CONFIG CORE ##
+#####################
+
+# Node database availability
```

Comment:
why `Node database availability` aren't param for orc/pg ?

Reply:
pg is a next PR